pynxtools/nexus/nexus.py

```diff
@@ -1,6 +1,5 @@
 # pylint: disable=too-many-lines
-"""Read files from different format and print it in a standard NeXus format
-"""
+"""Read files from different format and print it in a standard NeXus format"""
```

Comment:
I think the cleanest and most readable way to have this docstrings is the following:

```suggestion
"""
Read files from different format and print it in a standard NeXus format
"""
```

This is also how nomad does it.

Reply:
Now I see your point on this non-default ruff styling issue, I don't what is our consensus? @sanbrock others?